scripts/bountiful.ash

```diff
@@ -665,10 +671,15 @@ boolean hunt_bounty(bounty b) {
             mall_price(CONTENT_ITEMS[b.location]) <= maxSpecial)) {
     if(useBan)
       buy_banishers();
-
+    
+    //If have eagle and don't have RWB, use eagle; get those free copies.
+    if(have_familiar($familiar[Patriotic Eagle]) && !have_effect($effect[Everything Looks Red, White and Blue]))
+      use_familiar($familiar[Patriotic Eagle]);
+    else {
```

Comment:
make this an `else if` by moving line 680 up to here

Reply:
Updated